mautic.bc.yaml

```diff
@@ -10,16 +10,16 @@ objects:
       annotations:
         openshift.io/generated-by: OpenShiftNewBuild
       labels:
-        build: ${NAME}
-      name: ${NAME}
+        build: ${NAME}-${PR}-${SUFFIX}
```

Comment:
just ${NAME}${SUFFIX}. If you think of what a PR resemebles in the infra it really is just a suffix